docs/usermanual.md

```diff
@@ -98,17 +98,50 @@ hello
 
 then the resulting attribute sent to ContextBroker:
 
-{
-   ...
-   "attrHex": {
-     "value": "68656c6c6f"
-     "type": "<the one used at provisiong time for attrHex attribute>"
-   }
+{ ... "attrHex": { "value": "68656c6c6f" "type": "<the one used at provisiong time for attrHex attribute>" } }
+
+Note that every group of 2 character (I.E, the first group, `68`) corresponds to a single ASCII character or byte
+received in the payload (in this case, the value `0x68` corresponds to `h` in ASCII). You can use one of the multiple
+tools available online like [this one](https://string-functions.com/string-hex.aspx)
+
+##### NGSI-v2 and NGSI-LD Measure reporting
+
+It is possible report as a measure a NGSI-v2 or NGSI-LD payload when related device/group is configured with payloadType
+`ngsiv2` or `ngsild`. In these cases payload is ingested as measure where entity attributes are measure attributes and
+`id` and `type` are ignored, since `id` and `type` from device/group configuration provisioned are used.
+
+Examples of these ngsi payloads are:
+
+```
+ {
+     actionType: 'APPEND',
+     entities: [
+        {
+          id: 'MyEntityId1',
+          type: 'MyEntityType1',
+          attr1: { type: 'text', value: 'MyAttr1Value'},
+          ...
+        },
+        ...
+    ]
```

Comment:
```suggestion
     "actionType": "append",
     "entities": [
        {
          "id": "MyEntityId1",
          "type": "MyEntityType1",
          "attr1": { type: "Text", "value": "MyAttr1Value"},
          ...
        },
        ...
    ]
```

Note the change from APPEND to append, given the later is the right one in NGSI-v2 (uppercase variant is supported, but as a legacy from NGSI-v1). I understand it also works with lowercase, but please check.